### PR TITLE
[Tests] Add test for TimeAverageBrokerData

### PR DESCRIPTION
**Pull request by mattisonchao**
### Modifications

add test for TimeAverageBrokerData

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker;
+
+import org.apache.pulsar.policies.data.loadbalancer.NamespaceBundleStats;
+import org.junit.Test;
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java:22]: This is an invalid import in TestNG tests. Please use the TestNG Test annotation.

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        Set<String> bundles = new HashSet<>();
+        bundles.add("a");
+        Map<String, BundleData> emptyMap = new HashMap<>();
+        NamespaceBundleStats namespaceBundleStats = new NamespaceBundleStats();
+        namespaceBundleStats.msgThroughputIn = 1;
+        namespaceBundleStats.msgThroughputOut = 2;
+        namespaceBundleStats.msgRateIn = 3;
+        namespaceBundleStats.msgRateOut = 4;
+        timeAverageBrokerData.reset(bundles, emptyMap, namespaceBundleStats);
+        Assert.assertEquals(timeAverageBrokerData.getShortTermMsgThroughputIn(), 1);
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java:51]: using static import for Assert.assertEquals would improve readability of the test.

**Review by lhotari — CHANGES_REQUESTED** [review]

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker;
+
+import org.apache.pulsar.policies.data.loadbalancer.NamespaceBundleStats;
+import org.junit.Test;
+import org.testng.Assert;
+
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java:24]: Please use Pulsar formatting rules. The imports don't have empty lines between different types of imports. The instructions to setup code style / formatting rules is at https://github.com/apache/pulsar/#configure-code-style .

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java (hunk 1 above)
Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java]: Fixed :)

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java (hunk 2 above)
Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java]: Fixed :)

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java (hunk 3 above)
Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java]: Fixed :)

**Comment by mattisonchao** [comment]
Thank you for your rigorous review 👍🏼 , I've been fixing some problem. 
Could you review it again? @lhotari

**Review by lhotari — APPROVED** [review]

**Comment by mattisonchao** [comment]
Is the test failure caused by the environment? @lhotari

**Comment by mattisonchao** [comment]
Is the test failure caused by the environment? @merlimat

**Comment by eolivelli** [comment]
they are flaky tests. I restarted the jobs

**Comment by mattisonchao** [comment]
Fail again @eolivelli  :(

**Review by sijie — APPROVED** [review]

**Pull request merged**